backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -46,6 +46,7 @@ include::modules/oadp-configuring-noobaa-for-dr.adoc[leveloffset=+1]
 * link:https://{velero-domain}/docs/v{velero-version}/locations/[Overview of backup and snapshot locations in the Velero documentation]
 
 include::modules/about-oadp-update-channels.adoc[leveloffset=+1]
+include::modules/about-installing-oadp-on-multiple-namespaces.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
```

Comment:
Since this additional resources section is a trailing include, it should stay with the previous module.